### ISSUE TITLE
Feature: Better key error handling

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#892]
Report back if the number is incorrect on a key.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [2.0.29](https://github.com/open-wa/wa-automate-nodejs/compare/2.0.28...2.0.29) (2020-10-07)

- build(deps): bump ts-json-schema-generator from 0.74.0 to 0.75.0 [`#897`](https://github.com/open-wa/wa-automate-nodejs/pull/897)
- build(deps): bump puppeteer-extra-plugin-stealth from 2.6.1 to 2.6.2 [`#898`](https://github.com/open-wa/wa-automate-nodejs/pull/898)
- fixed #895 [`#895`](https://github.com/open-wa/wa-automate-nodejs/issues/895)
- fixed #900 [`#900`](https://github.com/open-wa/wa-automate-nodejs/issues/900)
- Release 2.0.29 [`d00a3b2`](https://github.com/open-wa/wa-automate-nodejs/commit/d00a3b257809a581458b02d3262dadfd883bf5a6)
- added pseudo code for testing framwork #896 [`f69e77c`](https://github.com/open-wa/wa-automate-nodejs/commit/f69e77c53a15fc06066b11954a46c2e90acc1a9e)
- Documentation improvements [`fc0b875`](https://github.com/open-wa/wa-automate-nodejs/commit/fc0b875bf0dabd99c258a6fa2ba74b4c85c7fe0c)
- Feature: Better key error handling #892 [`d732f8c`](https://github.com/open-wa/wa-automate-nodejs/commit/d732f8c3ca8b62e203aeaa31ef4f2f2d12bdef06)
